pyspecdata/datadir.py

```diff
@@ -371,6 +371,7 @@ def rclone_search(fname,exp_type,dirname):
                 remotelocation,
                 # dirname below needs to be replaced with path relative to current directory
                 os.path.normpath(os.path.join(dirname)).replace('\\','\\\\')))
+    cmd = cmd.replace("'",'"')
```

Comment:
You also want it to be clear how to use the rclone thing. There is already some documentation under search file name and find file that you could point to from the main readme file or from index.rst, as makes sense to you. 

But a big improvement that you can make here is that now you've been through the process of doing the rclone setup, would you be able to do that based on the documentation as it is now?

You can alsep search the directory for rclone to find existing documentation

Reply:
I don't think that this has been addressed

Reply:
added rclone note to README.rst